packages/labs/observers/README.md

```diff
@@ -43,6 +45,32 @@ class MyElement extends LitElement {
 }
 ```
 
+### ResizeController
+
+```ts
+import { ResizeController } from "@lit-labs/observers/resize-controller.js";
+// ...
+
+class MyElement extends LitElement {
+  private _resizeController = new ResizeController(this, {});
+
+  render() {
+    const width = this.getBoundingClientRect().width;
```

Comment:
I think the idea with `ResizeObserver` is that we can avoid calling `getBoundingClientRect`, which negatively affects performance due to the layout calculations it triggers.

The dimensions are already provided to `ResizeController`, and they can be extracted with something like this:


```typescript
private _resizeController = new ResizeController<DOMRectReadOnly>(this, {
    // Save into _resizeController.value only the current rect dimentions
    callback: (entries: ResizeObserverEntry[]) => entries[entries.length - 1].contentRect,
  });

render() {
  const width = this._resizeController.value.width;
  const height = this._resizeController.value.height;

  return html`<div class="container">${width}px x${height}px</div>`;
}
```

Reply:
Interesting 🤔 will definitely give this a go in my other project.  These are the sorts of examples/documentation that I'm missing I think to better use these packages

Reply:
> These are the sorts of examples/documentation that I'm missing I think to better use these packages

Agreed, though maybe a better default value for `callback` can also be provided? I would assume that most consumers would be interested in the current dimensions being exposed in `.value` (and not `undefined` as it is the case right now).

Reply:
For all my use-cases, I think this could be a better default. But can also imaging it could be confusing if some situation required getting the `borderBoxSize` or `contentBoxSize`.  Happy to test out any new changes that are made 🙏